docs/config.md

```diff
@@ -51,19 +51,22 @@ Supported keys include:
 
 | Key | Description | Supported Values | Default |
 | :--- | :--- | :--- | :--- |
-| `artifacts.taskrun.format` | The format to store `TaskRun` payloads in. | `in-toto`| `in-toto` |
+| `artifacts.taskrun.format` | The format to store `TaskRun` payloads in. | `in-toto`, `slsa/v1`| `tekton` |
```

Comment:
Is this right? Shouldn't the `default` value be one of the supported values? :thinking: 
(Same for the PipelineRun Configuration section below.)

Reply:
oh I see. Yes, you're right.